client/session.go

```diff
@@ -2672,8 +2873,10 @@ func (qs peerBlocksQueues) closeAll() {
 }
 
 type blocksMetadata struct {
-	peer   peer
-	id     ts.ID
+	peer peer
+	id   ts.ID
+	// TODO:
```

Comment:
suggestion(take/leave): change this to `TODO(user): message`, e.g. `TODO(rartoul): Make this not a slice once we delete the V1 code path`. It's a lot easier to parse the output of `grep -i TODO(rartoul)` if you follow that convention

Reply:
yeah totally fair, I thinking I was just following the convention I saw in some of the file I was working on. I'll update this

Reply:
done